src/tmplits/tmplits-hmilayout/module.js

```diff
@@ -191,38 +192,37 @@ class basicLayout extends HTMLElement {
             case 'value':
                 this.selectPage(newValue)
                 //Raise an event
-                if(!this.noChange){
+                if (!this.noChange) {
                     this.dispatchEvent(new Event("change", {
                         "bubbles": true,
                         "cancelable": true
-                    }));    
+                    }));
                 }
                 break;
             default:
                 break;
         }
     }
 
-    selectPage(templatesIndex){
-        let templates = this.shadowRoot.querySelectorAll('template')
-        let content = this.shadowRoot.querySelector('slot:not([name])')
-        // let footer = this.querySelector('[slot="footer"]')
-        
-        if(+templatesIndex < templates.length){            
-            let contentArea = this.querySelector('#contentArea')
-            contentArea.innerHTML = templates[templatesIndex].innerHTML
-            // content.innerHTML = templates[templatesIndex].innerHTML
-            // this.innerHTML = content.innerHTML
-            // this.append(footer)
+    selectPage(templatesIndex) {
+        let divs = this.querySelectorAll('div[slot]')
+        divs.forEach(div => {
+            // set every other slot div back to list slot
+            div.setAttribute('slot', 'list')
+            // TODO could do this more elegantly
+        });
+
+        if (+templatesIndex < divs.length) {
+            divs[templatesIndex].setAttribute('slot', 'content')
```

Comment:
The crux of this approach: setting this causes this div to be displayed inside <div class="tmplit-main-content">